src/lib.rs

```diff
@@ -568,6 +568,15 @@ impl Library {
     }
 
     fn parse_libs_cflags(&mut self, name: &str, output: &[u8], config: &Config) {
+        #[cfg(windows)]
+        lazy_static::lazy_static! {
+            static ref LIB_BASENAME: regex::Regex = regex::Regex::new(r"^(.*)\.(lib|dll)$").unwrap();
```

Comment:
On mingw AFAIK there are also `.a` static libraries

Reply:
Do they have the `lib` prefix?

Reply:
IIRC on mingw the libraries all have `lib` as prefix, like on Linux. But I'm not 100% sure

Reply:
Checking this again, on Windows you'll either link to a `foo.lib` (import library, can be static or not) or a `libfoo.a` (mingw static library). It can theoretically also be a `libfoo.dll` but that's only supported with mingw and generally a mistake.

Reply:
@Be-ing Do you want to update the PR accordingly?

Reply:
Yes, thanks for the reminder.

Reply:
Hmm, so how to handle this? Should there be a separate regex for mingw vs msvc?

Reply:
I'd say so, yes

Reply:
Okay, I added a separate regex for MinGW. I am not very familiar with MinGW, so I'm just going on what you said above. I have not tested with MinGW.

Reply:
Thanks!